src/lib/PermissionStore.ts

```diff
@@ -0,0 +1,165 @@
+export interface Permission {
+    origin: string;
+    allowsAll: boolean;
+    addresses: string[];
```

Comment:
I think we should leverage the advantages of a typed language and use `Nimiq.Address` here.